### ISSUE TITLE
is this GEM support UUID ?

### ISSUE THREAD

**Issue by buncismamen** [LTe/acts-as-messageable#107]
because I got this error

on rails 6, using uuid and postgresql 

```
irb(main):018:0> user.messages
  ActsAsMessageable::Message Load (3.3ms)  SELECT "messages".* FROM "messages" WHERE ((sent_messageable_type = 'User' and
                        sent_messageable_id = '33af867e-1e5a-4d24-8c85-337b485d3a72' and
                        sender_delete = FALSE and sender_permanent_delete = FALSE) or
                        (received_messageable_type = 'User' and
                        received_messageable_id = '33af867e-1e5a-4d24-8c85-337b485d3a72' and
                        recipient_delete = FALSE and recipient_permanent_delete = FALSE)) ORDER BY created_at desc LIMIT $1  [["LIMIT", 11]]
Traceback (most recent call last):
ActiveRecord::StatementInvalid (PG::InvalidTextRepresentation: ERROR:  invalid input syntax for integer: "33af867e-1e5a-4d24-8c85-337b485d3a72")
LINE 2:                         sent_messageable_id = '33af867e-1e5a...
                                                      ^

````

**Comment by LTe** [comment]
Hi @buncismamen thanks for bug report. Right now `ActsAsMessageable::Message` does not support `UUID` as foreign key. I will take a look and I will see what we can do about this.

**Comment by LTe** [comment]
@buncismamen I just released `0.5.0` version with UUID support. You can check documentation about this here https://github.com/LTe/acts-as-messageable#post-installation
